.github/workflows/release.yml

```
@@ -0,0 +1,32 @@
+name: Keycloak Client Nightly Release
+
+on:
+  schedule:
+    - cron: '0 2 * * *'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 17
```

We're using 21 for the other workflow, let's keep that aligned here, or downgrade the version 17 in both.
```suggestion
          java-version: 21
```

---

Then let's downgrade the CI workflow as well.